input/docs/getting-started/running-cake-scripts.md

```diff
@@ -0,0 +1,61 @@
+Order: 10
+---
+
+There are different runners available for running Cake scripts.
+
+# .NET Core Tool
+
+:::{.alert .alert-info}
+This is the recommended way to run Cake Scripts.
+:::
+
+The [Cake.Tool](https://www.nuget.org/packages/Cake.Tool) NuGet package, is a .NET Core global tool compiled for .NET Core 2.1 or newer.
+
+:::{.alert .alert-info}
+The following instructions require .NET Core 3.0 or newer.
+See [How to manage .NET Core tools](https://docs.microsoft.com/en-us/dotnet/core/tools/global-tools) for details and other options.
+:::
+
+Make sure to have a tool manifest available in your repository or create one using the following command:
+
+```shell
+dotnet new tool-manifest
+```
+
+Install Cake as a local tool using the `dotnet tool` command:
+
+```shell
+dotnet tool install Cake.Tool --version x.y.z
+```
+
+Once installed, you can launch Cake from anywhere using the .NET CLI:
+
+```shell
+dotnet cake
+```
+
+:::{.alert .alert-info}
+By convention this will execute the build script named `build.cake`.
+You can override this behavior by additionally passing the name of the build script.
+:::
+
+# Cake runner for .NET Framework
+
+The [Cake](https://www.nuget.org/packages/Cake) NuGet package is a runner requiring [.NET Framework 4.6.1](https://www.microsoft.com/net/download/dotnet-framework/net461)
+or newer on Windows and Mono `5.12.0` or newer on Mac or Linux.
+
+:::{.alert .alert-warning}
+It is suggested to use [.NET Framework 4.7.2](https://www.microsoft.com/net/download/dotnet-framework/net472) or newer to run build scripts
+which are using addins targeting .NET Standard 2.0 only.
+:::
+
+The runner is mainly for backwards compatibility where scripts or addins are used which require .NET Framework.
+In all other cases it is recommended to use [.NET Core Tool](#net-core-tool).
+
+# Cake runner for .NET Core
+
+The [Cake.CoreCLR](https://www.nuget.org/packages/Cake.CoreCLR) NuGet package is a runner compiled for .NET Core 2.0.
```

Comment:
Again, is it worth mentioning "how" this can be used?  How to get the bits, how to invoke it, etc?

Reply:
Is there still a valid use case for .NET Core runner which justifies to have further documentation? In the current version I declared it as deprecated with .NET Core Tool as replacement.

Reply:
I would also declare it "legacy", so think the alert below is enough. Don't think we need to promote it's usage.

Reply:
At the minute, I believe I am right in saying that Cake.CoreClr is the default usage for Cake.Bakery within the VSCode Extension for Cake.

Reply:
But it should be able to use Cake.Tool right @mholo65?

Reply:
I think it "should" be able to, yes, but currently it isn't.  Perhaps part of this work should be to update the extensions default usage as well.

Reply:
@gep13 
> I think it "should" be able to, yes, but currently it isn't.

I'm using global Cake.Tool all the time, and have no problem with it. What issues are you seeing? The one thing I know is a problem, is using Cake.exe via Choco, due to the shim that Choco creates. E.g. Cake.Core.dll is not next to Cake.exe.

Reply:
@mholo65 I am not seeing any issues.  What I am saying is that I believe, unless it has changed, that the default use of Cake.Bakery from the extension is to use Cake.CoreClr.

Reply:
Closing this here since https://github.com/cake-build/cake-vscode/issues/376 was created for updating the Visual Studio Code extension.